packages/dds/tree/src/simple-tree/core/unhydratedFlexTree.ts

```diff
@@ -150,11 +150,18 @@ export class UnhydratedFlexTreeNode implements UnhydratedFlexTreeNode {
 	public adoptBy(parent: UnhydratedFlexTreeField, index: number): void;
 	public adoptBy(parent: UnhydratedFlexTreeField | undefined, index?: number): void {
 		if (parent !== undefined) {
-			assert(
-				this.location === unparentedLocation,
-				0x98c /* Node may not be adopted if it already has a parent */,
-			);
 			assert(index !== undefined, 0xa08 /* Expected index */);
+			if (this.location !== unparentedLocation) {
+				throw new UsageError("A node may not be in more than one place in the tree");
```

Comment:
We can have many trees, especially unhydrated ones.
```suggestion
				throw new UsageError("A node may not be in more than one place in a tree or in more than one tree");
```